hardware_interface/include/hardware_interface/types/hardware_interface_type_values.hpp

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 ros2_control Development Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef HARDWARE_INTERFACE__TYPES__HARDWARE_INTERFACE_TYPE_VALUES_HPP_
+#define HARDWARE_INTERFACE__TYPES__HARDWARE_INTERFACE_TYPE_VALUES_HPP_
+
+namespace hardware_interface
+{
+constexpr const auto HW_IF_POSITION = "position";
+constexpr const auto HW_IF_VELOCITY = "velocity";
+constexpr const auto HW_IF_EFFORT = "effort";
```

Comment:
why do we need these values? I thought they'd be defined as constants in the control_msgs?

Reply:
This is not relevant in #121 anymore.

Reply:
what does that mean for this PR?